editors/code/src/lsp_ext.ts

```diff
@@ -113,3 +113,14 @@ export interface CommandLinkGroup {
     title?: string;
     commands: CommandLink[];
 }
+
+export interface DocumentationLink {
+    remote: string;
```

Comment:
`lc.Url`, if they have such type.

Reply:
They don't. There's `lc.URI` but I think that's intended for a different purpose.